lib/tapioca/compilers/dsl/action_controller_helpers.rb

```diff
@@ -76,6 +76,8 @@ class ActionControllerHelpers < Base
             .void
         end
         def decorate(root, constant)
+          return unless constant.name
```

Comment:
Can I suggest that we instead reject these constants at `gather` phase?

I am thinking something like:
```
::ActionController::Base.descendants.reject(&:abstract?).select(&:name)
```

We try to do most high-level filtering in `gather` phase and only do short-circuiting in `decorate` for structural checks.